lib/Request.js

```diff
@@ -155,10 +154,10 @@ class Request {
    * Promise).
    */
 
-  each(func, isAsync) {
+  each(func, isAsync, concurrency) {
```

Comment:
the concurrency can be set for any function other than `forEach`, so this seems unnecessary to me. Why not stick with the original code for consistency?